api/openapi-spec/openapi.yaml

```diff
@@ -470,7 +526,7 @@ components:
         type:
           type: string
         value:
-          type: object
+          type: string
```

Comment:
why this change? JSON type?

Reply:
It is of type `string` in go-sdk. Guess it was wrong earlier. https://github.com/optimizely/go-sdk/blob/master/pkg/config/optimizely_config.go#L106